pyproject.toml

```diff
@@ -26,7 +26,7 @@ test-command = "pytest -s --verbose {package}/tests"
 test-extras = ["dev"]
 
 before-all = [
-    "git clone --branch master --depth 1 https://github.com/eclipse-cyclonedds/cyclonedds.git main",
+    "git clone --branch releases/0.10.x --depth 1 https://github.com/eclipse-cyclonedds/cyclonedds.git",
```

Comment:
We mustn't change the core `cyclonedds` repo branch to `releases/0.10.x` on the `master` branch of the Python binding!

Reply:
I think that we needed to add version 0.10.2  for cyclonedds-python, but there was a change in the main branch of C development. In fact, it is already a new version, so I am a bit confused about what to do

Reply:
I saw a bug in the main branch of Cyclonedds C version 0.10.3, which caused the build to fail, see https://dev.azure.com/eclipse-cyclonedds/cyclonedds-python/_build/results?buildId=6119&view=results

Reply:
It is possible to update and wait for a stable version of 0.10.3 before building a more secure one

Reply:
@eboasson hi, I build python 3.11+ version for cyclonedds-python3.10.2、cyclonedds3.10.2, Can you confirm and upload it to the pypi library? The build script is open source, and the entire execution is in the github container. I tested that it is available on both windows and linux.

https://github.com/LY1806620741/unitree_sdk2_python/actions/runs/9350099214